src/iexamine.cpp

```diff
@@ -1716,43 +1716,7 @@ void iexamine::pit_covered( Character &you, const tripoint &examp )
 /**
  * Loop prompt to bet $10.
  */
```

Comment:
```suggestion
```
Removing this as it applies to what you are removing and not the 2x4 covered pit above.